nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/features/PlatformFunctions.java

```diff
@@ -263,4 +265,11 @@ public boolean documentExists(CoreSession session, String idOrPath) {
         return session.exists(documentRef);
     }
 
+    /**
+     * @since 11.1
+     */
+    public DateWrapper calendar(Date inputDate) {
```

Comment:
Looking at the whole code, I've put this method in `CoreFunctions` with other `date` and `calendar` methods.